jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/Response.java

```diff
@@ -38,22 +38,34 @@ public interface Response
     /**
      * @return the request associated with this response
      */
-    Request getRequest();
+    default Request getRequest()
```

Comment:
The javadoc for this method has to clearly define what "associated" means.  Is it the original request, or the request in the conversation that triggered this response?